packages/wonder-blocks-testing/src/fixtures/combine.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import {clone} from "@khanacademy/wonder-stuff-core";
+
+/**
+ * Combine two values.
+ *
+ * This method clones val2 before using any of its properties to try to ensure
+ * the combined object is not linked back to the original.
+ *
+ * If the values are objects, it will merge them at the top level, with val2
+ * overwriting val1 where there are conflicts.
+ *
+ * If the values are arrays, it will concatenate and dedupe them.
+ * NOTE: duplicates in either val1 or val2 will also be deduped.
+ *
+ * If the values are any other type, or val2 has a different type to val1, val2
+ * will be returned.
+ */
+export const combine = (val1: $ReadOnly<any>, val2: $ReadOnly<any>): any => {
```

Comment:
There's a number of different npm packages that do deep object merging. Could we have used one of those?

Reply:
This doesn't do a deep merge and I don't think we want it to.

Reply:
Not sure why I thought it was a deep merge.😳

Reply:
I'll rename the function to clear this up a bit.